beacon-chain/core/state/benchmarks/README.md

```diff
@@ -16,10 +16,36 @@ go run beacon-chain/core/state/benchmarks/benchmark_files/generate_bench_files.g
 
 Bazel does not allow writing to the project directory, so running with `go run` is needed.
 
+## Running the benchmarks
+To run the ExecuteStateTransition benchmark:
+
+```bazel test //beacon-chain/core/state/benchmarks:go_default_test --test_filter=BenchmarkExecuteStateTransition_FullBlock --test_arg=-test.bench=BenchmarkExecuteStateTransition_FullBlock```
+
+To run the ExecuteStateTransition (with cache) benchmark:
+
+```bazel test //beacon-chain/core/state/benchmarks:go_default_test --test_filter=BenchmarkExecuteStateTransition_WithCache --test_arg=-test.bench=BenchmarkExecuteStateTransition_WithCache```
+
+To run the ProcessEpoch benchmark:
+
+```bazel test //beacon-chain/core/state/benchmarks:go_default_test --test_filter=BenchmarkProcessEpoch_2FullEpochs --test_arg=-test.bench=BenchmarkProcessEpoch_2FullEpochs```
+
+To run the HashTreeRoot benchmark:
+
+```bazel test //beacon-chain/core/state/benchmarks:go_default_test --test_filter=BenchmarkHashTreeRoot_FullState --test_arg=-test.bench=BenchmarkHashTreeRoot_FullState```
+
+To run the HashTreeRootState benchmark:
+
+```bazel test //beacon-chain/core/state/benchmarks:go_default_test --test_filter=BenchmarkHashTreeRootState_FullState --test_arg=-test.bench=BenchmarkHashTreeRootState_FullState```
+
+Extra flags needed to benchmark properly:
+
+```--nocache_test_results --test_arg=-test.v --test_timeout=2000 --test_arg=-test.cpuprofile=/tmp/cpu.profile --test_arg=-test.memprofile=/tmp/mem.profile --test_output=streamed```
```

Comment:
Split this up so it wouldn't be duplicated so many times, if you want me to group these flags with the commands themselves let me know!